share/metainfo/io.github.yuxshao.ptcollab.metainfo.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.github.yuxshao.ptcollab</id>
+  
+  <name>pxtone Collab</name>
+  <summary>Sample-based music editor where you can collaborate with friends!</summary>
+  
+  <metadata_license>MIT</metadata_license>
+  <project_license>MIT</project_license>
+  
+  <description>
+    <p>
+      Unofficial flatpak package of pxtone Collab
+    </p>
+    <p>
+      About: pxtone collab (ptcollab) is a sample-based music editor where you can collaborate with friends! It&apos;s based off the music engine of pxtone, a music editor developed by Pixel, the creator of Cave Story and Kero Blaster.
+    </p>
+    <p>
+      Features: ptcollab has many standard features you might expect from a music editor. If you&apos;ve used pxtone collage before, the following will be familiar:
+    </p>
+    <ul>
+      <li>Note and parameter input (pan, volume, portamento, fine-tuning, effects)</li>
+      <li>Instrument import (ptvoice, ogg, wav, ptnoise)</li>
+      <li>Song parameter settings (tempo, time signature)</li>
+	</ul>
+	<p>
+      In addition, ptcollab supports:
+    </p>
+    <ul>
+      <li>Collaborative editing: host a server and have others connect and work with you!</li>
+	  <li>Session recording: save a frame-by-frame recording of a session and play it back later!</li>
+      <li>MIDI input: Record notes either live while the song is playing, or using keyboard controls to step through time.</li>
+    </ul>
+  </description>
+  <launchable type="desktop-id">io.github.yuxshao.ptcollab.desktop</launchable>
+  <content_rating type="oars-1.1">
+    <content_attribute id="social-chat">mild</content_attribute>
+  </content_rating>
```

Comment:
missing `releases` and `screenshots`

Reply:
screenshots added!
I'm trying to figure out how releases work

Reply:
done both